apps/meteor/tests/end-to-end/api/livechat/04-dashboards.ts

```diff
@@ -984,6 +984,60 @@ describe('LIVECHAT - dashboards', function () {
 			expect(originalFirstResponseTimeInSeconds).to.be.greaterThanOrEqual(firstDelayInSeconds);
 		});
 
+		it('should correctly associate the first response time to the first agent who responded the room', async () => {
+			const response = await startANewLivechatRoomAndTakeIt({ agent: forwardAgent.credentials });
+			roomId = response.room._id;
+
+			await sendAgentMessage(roomId, 'first response from agent', forwardAgent.credentials);
+
+			await request
```

Comment:
Shouldn't we send a message with the new agent just to ensure it?

Reply:
Hmmm good idea, done!
This issue should happen even if the user who received the room didn't send any new message, but it's good to have this covered